src/mwoffliner.lib.ts

```diff
@@ -25,7 +25,7 @@ import Redis from './Redis';
 import { writeFilePromise, mkdirPromise, isValidEmail, genHeaderCSSLink, genHeaderScript, genCanonicalLink, saveStaticFiles, readFilePromise, makeArticleImageTile, makeArticleListItem, getDumps, getMediaBase, MIN_IMAGE_THRESHOLD_ARTICLELIST_PAGE, downloadAndSaveModule, getSizeFromUrl, getRelativeFilePath } from './util';
 import { mapLimit } from 'promiso';
 import logger from './Logger';
-import { getAndProcessStylesheets } from './util';
+import { getAndProcessStylesheets, getCustomProcessorPath } from './util';
```

Comment:
Naming is wrong, you mean `getCustomFlavourPath`? secure the naming of variable is proper everywhere in your PR.